server/services/licenceService.ts

```diff
@@ -11,6 +12,44 @@ export default class LicenceService {
     return new LicenceApiClient(token).getTestData()
   }
 
+  async createLicence(username: string): Promise<CreateLicenceResponse> {
+    const token = await this.hmppsAuthClient.getSystemClientToken(username)
+    // TODO construct with real licence data using prison and community APIs
+    const licence = {
+      typeCode: 'AP',
+      version: '1.0',
+      nomsId: Math.floor(Math.random() * 900000 + 100).toString(), // Generate random nomsId for now ... should be unique ID from nomis later
+      bookingNo: '12334',
```

Comment:
We could use a real nomsId that we know is in DEV? e.g. This one is in both Nomis and Delius in the DEV env - `A5198DY`

Reply:
Reason for this randomness is the API rejects duplicate nomsIds, and since this is stubbed data we would have to delete it each time from the database when clicking through the flow. Hence this is randomised for now while we have the stubbed data